src/libecalc/presentation/yaml/yaml_types/models/yaml_compressor_chart.py

```diff
@@ -76,6 +76,25 @@ def to_dto(self):
         raise NotImplementedError
 
 
+class YamlSingleSpeedChart(YamlBase):
+    name: str = Field(
+        ...,
+        description="Name of the model. See documentation for more information.",
+        title="NAME",
+    )
+    type: Literal[YamlModelType.COMPRESSOR_CHART] = Field(
+        YamlModelType.COMPRESSOR_CHART,
+        description="Defines the type of model. See documentation for more information.",
+        title="TYPE",
+    )
+    chart_type: Literal[YamlChartType.VARIABLE_SPEED] = YamlChartType.VARIABLE_SPEED
+    curves: YamlCurve = Field(..., description="One single compressor chart curve.", title="CURVE")
+    units: YamlUnits = None
+
+    def to_dto(self):
+        raise NotImplementedError
```

Comment:
I think it is a mistake to define this as needed in YamlBase, as I have done currently.

Reply:
no, it doesnt matter that much. We want to skip `dtos` completely, and map directly to domain. dtos are no longer needed, but we will need some clever ways to map to domain. WIP :)